bundles/pubsub/pubsub_admin_zmq/src/pubsub_zmq_topic_sender.c

```diff
@@ -621,8 +626,9 @@ static int psa_zmq_topicPublicationSend(void* handle, unsigned int msgTypeId, co
 
                     //send Payload
                     if (rc > 0) {
+                        int flag = ((metadataLength > 0)  && (footerLength > 0)) ? ZMQ_SNDMORE : 0;
```

Comment:
Shouldn't this be an OR "||'?

Reply:
Correct I have fixed this and added test

Reply:
You are correct a found this error also.
 I have corrected this and added test for this